keps/sig-node/3857-rro-mounts/kep.yaml

```diff
@@ -6,7 +6,7 @@ owning-sig: sig-node
 #participating-sigs:
 #  - sig-aaa
 #  - sig-bbb
-status: implementable
+status: implemented
```

Comment:
(updated this line in the latest edit)

Reply:
Don't change it, implemented is reserved only after you GA.

Reply:
Fixed